geode-core/src/test/java/org/apache/geode/internal/JarDeployerIntegrationTest.java

```diff
@@ -126,14 +111,16 @@ public void testDeployToInvalidDirectory() throws IOException, ClassNotFoundExce
       public void run() {
         try {
           barrier.await();
-        } catch (Exception e) {
-          fail(e);
+        } catch (InterruptedException iex) {
```

Comment:
This wasn't a test I wrote, it's been around for awhile.  I pulled it out of a DUnit into this class in 917dfa0.  It wasn't actually clear to me what the author was trying to test here, so I left it around intending to come back to it later.  Do you have any idea of what the author was trying to test?  The name of the test (testDeployToInvalidDirectory) seems unrelated to Concurrency.